app/src/main/java/org/fossasia/openevent/app/data/EventsDataRepository.java

```diff
@@ -109,13 +115,29 @@ private void saveEvents(List<Event> events) {
         setupEventService();
 
         return eventService
-            .getEvents(authorization)
-            .doOnNext(this::saveEvents)
-            .subscribeOn(Schedulers.io())
-            .observeOn(AndroidSchedulers.mainThread());
+                .getEvents(authorization)
+                .doOnNext(this::saveEvents)
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread());
     }
 
-    public void setEventService(EventService eventService) {
-        this.eventService = eventService;
+    @Override
+    public Observable<LoginResponse> login(String username, String password) {
```

Comment:
Well then it is not event repository, and will increase in size very quickly

We want to keep classes well defined and doing only one kind of work

I was even thinking of creating different repository for attendees, so that they remain small and testable